elm/Main.elm

```diff
@@ -589,11 +589,12 @@ cmdPlaylistAdd id =
     buildWsCmdID "add" id
 
 
-cmdSeek : Float -> String
-cmdSeek seconds =
+cmdSeek : String -> Float -> String
+cmdSeek id seconds =
     Encode.encode 0 <|
         Encode.object
             [ ( "cmd", Encode.string "seek" )
+            , ( "id", Encode.string id )
```

Comment:
`id` is always used as an opaque id which is echoed back in the reply. Maybe add some field just for this? Say, `song` or `songid`?

Reply:
go ahead

don't think I'm going to touch this soon